torch/fx/passes/dialect/common/cse_pass.py

```diff
@@ -0,0 +1,90 @@
+from typing import Dict, Tuple, Any
+
+import torch
+import torch.fx as fx
+from torch.fx.passes.infra.pass_base import PassBase, PassResult
+from torch.utils._pytree import tree_flatten
+
+aten = torch.ops.aten
+
+
+def get_aten_target(node):
+    if hasattr(node.target, 'overloadpacket'):
+        return node.target.overloadpacket
+    return node.target
+
+# random ops are banned from CSE
+rand_ops = [aten.dropout, aten._fused_dropout, aten._standard_gamma,
```

Comment:
what about prim.uniform ?

To make this pass general and dialect agnostic, the pass need to accept this list as an input.